api/proto/info/info.proto

```diff
@@ -183,6 +183,18 @@ enum HwSecurityModuleStatus {
     ENABLED  = 3;  //HSM is found and being actively used
 }
 
+enum DataSecAtRestStatus {
+    DATASEC_AT_REST_UNKNOWN  = 0; // Status is unknown
+    DATASEC_AT_REST_DISABLED = 1; // Enabled, but not being used
+    DATASEC_AT_REST_ENABLED = 2;  // Enabled, and used
+    DATASEC_AT_REST_ERROR = 4;    // Enabled, but encountered an error
+}
+
+message DataSecAtRest {
+    DataSecAtRestStatus status = 1; // Current status
+    string info = 2;                // Informational string augmenting status
```

Comment:
Do you expect this to be used for anything but errors?

For errors we have an existing (errorInfo?) message which has a string and a timestamp for when the error occurred. Don't know if that would make sense here.

Reply:
Hi Erik,
I am planning to have this field as a more verbose form of the state. i.e. even if this is enabled/disabled state, we can add more info in the form a string. E.g. if enabled, then give the list of default vaults etc..

Thanks
Hari

Reply:
e.g. if disk encryption feature is available but not being used (Not an error exactly), we can give a more verbose string like "file system incompatibility - needs disruptive usb upgrade".

Reply:
OK, there might be some UX questions around that when it comes to localization, but we can add a detailed enum later if that makes L10N easier.